### PR TITLE
Fixed country name and currency code for Belarus

### PR DESCRIPTION
**Pull request by dizel-by**

### CODE REVIEW

**Comment by mledoze** [comment]
@dizel-by thank you for this, your help is appreciated 👍

**Comment by herrjemand** [comment]
@mledoze This review thing is weird. I forgot to press submit the changes

File: countries.json [hunk 1]
```diff
@@ -1307,10 +1307,10 @@
 		"ccn3": "112",
 		"cca3": "BLR",
 		"cioc": "BLR",
-		"currency": ["BYR"],
+		"currency": ["BYN"],
 		"callingCode": ["375"],
 		"capital": "Minsk",
-		"altSpellings": ["BY", "Bielaru\u015b", "Republic of Belarus", "\u0411\u0435\u043b\u043e\u0440\u0443\u0441\u0441\u0438\u044f", "\u0420\u0435\u0441\u043f\u0443\u0431\u043b\u0438\u043a\u0430 \u0411\u0435\u043b\u0430\u0440\u0443\u0441\u044c", "Belorussiya", "Respublika Belarus\u2019"],
+		"altSpellings": ["BY", "Bielaru\u015b", "Republic of Belarus", "\u0411\u0435\u043b\u0430\u0440\u0443\u0441\u044c", "\u0420\u0435\u0441\u043f\u0443\u0431\u043b\u0438\u043a\u0430 \u0411\u0435\u043b\u0430\u0440\u0443\u0441\u044c"],
```
Comment (dizel-by) [inline countries.json]: "Белоруссия" is the name of former soviet republic. It is completely wrong to use it for Republic of Belarus during last 25 years.

**Comment by mledoze** [comment]
This issue has been resolved in https://github.com/mledoze/countries/commit/0d449e4a2de2851d27507b991d4e17c367553fe8

I removed the Belarus name in `altSpellings` and put Belorussia (in russian) instead, as @herrjemand suggested.

Thank you all for your help 👍.

**Pull request merged**